flask_mongorest/resources.py

```diff
@@ -344,6 +357,14 @@ def value_for_field(self, obj, field):
         raise UnknownFieldError
 
     def validate_request(self, obj=None):
+        if (request.method == 'PUT' and obj) or request.method == 'POST':
```

Comment:
We have mass-updating when you specify a filter and PUT some fields (used in tasks). For those, validate_request is called once for request data on PUT